plugins/kubernetes/src/utils/clusterLinks/formatters/index.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ClusterLinksFormatter } from '../../../types/types';
+import { standardFormatter } from './standard';
+import { rancherFormatter } from './rancher';
+import { openshiftFormatter } from './openshift';
+import { aksFormatter } from './aks';
+import { eksFormatter } from './eks';
+import { gkeFormatter } from './gke';
+
+export const clusterLinksFormatters: Record<string, ClusterLinksFormatter> = {
+  standard: standardFormatter,
+  rancher: rancherFormatter,
+  openshift: openshiftFormatter,
+  aks: aksFormatter,
+  eks: eksFormatter,
+  gke: gkeFormatter,
```

Comment:
should we really have all these that actually aren't implemented? sounds like it just risks leading to confusion

Reply:
I changed the error message for: "F formatter is not yet implemented. **Please, contribute**!"
I guess that it should help! 
I also think that it would help future implementers and make sure that the documentation is already up to date...
Finally, I'm gonna try to implement a couple of them...